src/cli/commands/config.mjs

```diff
@@ -28,6 +31,26 @@ export const builder = (yargs) => {
     .example(t('CMD_CONFIG_EXAMPLE3'), t('CMD_CONFIG_EXAMPLE3_DESC'));
 };
 
+const getDownloaders = () => {
+  const thisFileDir = path.dirname(fileURLToPath(import.meta.url));
+  const downloadersDir = path.join(thisFileDir, '..', '..', 'downloaders');
+  const downloaders = fs.readdirSync(downloadersDir).map((fullName) => path.parse(fullName).name);
```

Comment:
Do not use `fs` sync API

Reply:
Fixed in [b3c710d](https://github.com/FlandreDaisuki/dmhy-subscribe/pull/58/commits/b3c710d05f2ea783b14948d6f2bd8682a4e1ee1f)